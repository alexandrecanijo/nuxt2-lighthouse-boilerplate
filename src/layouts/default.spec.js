import { shallowMount, createLocalVue } from '@vue/test-utils';
import DefaultLayout from './default.vue';

describe('Default Layout', () => {
  const localVue = createLocalVue();

  const input = {
    localVue,
    mocks: {},
    stubs: {
      Nuxt: true,
    },
  };

  it('renders the layout', () => {
    const wrapper = shallowMount(DefaultLayout, input);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
