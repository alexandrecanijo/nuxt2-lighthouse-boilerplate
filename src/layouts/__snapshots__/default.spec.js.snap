// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default Layout renders the layout 1`] = `
"<main>
  <nuxt-stub></nuxt-stub>
</main>"
`;
